src/code42cli/cmds/securitydata.py

```diff
@@ -4,6 +4,8 @@
 import py42.sdk.queries.fileevents.filters as f
 from c42eventextractor.extractors import FileEventExtractor
 from click import echo
+from py42.sdk.queries.fileevents.filters import RiskIndicator
```

Comment:
I'm getting red squigglies for these two imports in PyCharm. I think that since these classes are exposed in py42/sdk/queries/fileevents/filters/__init__.py using this statement: `from py42.sdk.queries.fileevents.filters.risk_filter import *` , we want to import them here like `from py42.sdk.queries.fileevents.filters.risk_filter import RiskIndicator`.

Reply:
That statement is actually what allows these two lines to work. That's done for user convenience so that a user who is making a file event query doesn't need to have multiple imports if their queries span multiple categories of filters. If this was a problem, the unit tests wouldn't pass.

That being said, for usage that's internal like this, it might be easier for c42 developer to discover the source of a filter if we write this the way @peterbriggs42 is suggesting, especially since it's more consistent with the way other filters are already written.

Reply:
I just switched them, since it sounds like that would be more consistent? Is there a situation where I should leave it as `py42.sdk.queries.fileevents.filters` ?